components/wb-actionmng/actionmng.js

```diff
@@ -430,6 +430,8 @@ var $document = wb.doc,
 				input.setAttribute( "checked", true );
 			}
 		} );
+		const tagfilterContainer = sourceElm.closest( ".wb-tagfilter" ) || sourceElm;
+		$( tagfilterContainer ).trigger( "wb-contentupdated" );
```

Comment:
No need to get the tagfilter element. The tagfilter update is triggered if any child element sends the "wb-contentupdated" event.
```suggestion

		$( sourceElm ).trigger( "wb-contentupdated" );
```